### ISSUE TITLE
Custom HTML sanitation precision

### ISSUE THREAD

**Issue by j2l** [saltcorn/saltcorn#907]
Hello,
I'd need some external code to run on every page (using bootstrap CSS) and I pasted it into custom html field.
Some div class are simply removed while others are kept.
For example:
```
<nav class="navbar navbar-inverse bg-inverse fixed-top bg-faded">
  <div class="modal-dialog modal-lg" role="document">
    <div class="modal-content">
      <div class="modal-header">
```
is sanitized to
```
<nav>
  <div>
    <div>
      <div>
```

but `<button type='button' class='close' data-dismiss='modal' aria-label='Close'>` is kept as is.

Any idea why?
Thanks

**Comment by j2l** [comment]
Sure!
It is code from a working html page

**Comment by glutamate** [comment]
fixed in #908

**Comment by j2l** [comment]
Thanks!
